Header.h

```diff
@@ -0,0 +1,146 @@
+#pragma once
+#include <iostream>
+#include <string>
+using namespace std;
+class Movie
+{
+public:
+    int id;
+    string title;
+    string ranking;
+    int release_date;
+    int number_of_tickets;
+    int ticket_price_in_uan;
+    string comment;
+    int GetID(){
```

Comment:
Accessors and mutators (get and set functions) should be named like variables. GetID should be get_id